xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextGenerate.xtend

```diff
@@ -94,7 +97,10 @@ class XtextGenerate extends DefaultTask {
 			projectDir = project.projectDir
 			containerHandle = this.containerHandle
 			allFiles = allSources.files
-			allClasspathEntries = this.getNullSafeClasspath.files
+			val cp = this.getClasspath
+			if (cp !== null) {
```

Comment:
what about the old fallback here? did it ever make sense?

Reply:
Probably not.